aio/content/examples/form-validation/src/app/reactive/hero-form-reactive.component.2.ts

```diff
@@ -19,16 +19,21 @@ export class HeroFormReactiveComponent implements OnInit {
   heroForm: FormGroup;
 
   ngOnInit(): void {
+    // #docregion async-validator-usage
+    const alterEgoControl = new FormControl('', {
+      asyncValidators: [this.alterEgoValidator.validate.bind(this.alterEgoValidator)],
+      updateOn: 'blur'
+    });
+    alterEgoControl.setValue(this.hero.alterEgo);
+    // #enddocregion async-validator-usage
```

Comment:
I think we should exclude the `setValue` call from the doc region:
```suggestion
    // #enddocregion async-validator-usage
     alterEgoControl.setValue(this.hero.alterEgo);
```